source/wrangling.md

```diff
@@ -841,17 +841,28 @@ indicating they are integer data types (i.e., numbers)!
 
 Now that the `tidy_lang` data is indeed *tidy*, we can start manipulating it
 using the powerful suite of functions from the `pandas`.
-We revisit the `[]` from the chapter on {ref}`intro`,
-which lets us create a subset of rows from a data frame.
-Recall the argument to `[]`:
-a list of column names, or a logical statement that evaluates to either `True` or `False`,
-where `[]` returns the rows where the logical statement evaluates to `True`.
-This section will highlight more advanced usage of the `[]` function.
-In particular, this section provides an in-depth treatment of the variety of logical statements
+We will first revisit the `[]` from the chapter on {ref}`intro`,
+which lets us obtain a subset of either the rows **or** the columns of a data frame.
+This section will highlight more advanced usage of the `[]` function,
+including an in-depth treatment of the variety of logical statements
 one can use in the `[]` to select subsets of rows.
 
 +++
 
+### Extracting columns by name
```

Comment:
This is essentially a repeat of material from Ch1. But we do the same in the R version, and it doesn't hurt to have it here, especially if someone is going to search for how to subset rows/cols and end up in Ch 3 (maybe missing Ch 1 entirely). The section is named "extract rows or columns", so a bit odd not to discuss columns at all.